dfastmi/gui.py

```diff
@@ -28,19 +28,19 @@
 """
```

Comment:
Should we ask Bert to do some overall gui tests for this issue?

Reply:
Yes, let's ask Bert for a walkthrough of the GUI